Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -163,6 +163,13 @@ extension BrowserViewController: WKNavigationDelegate {
         info["title"] = webView.title
 
         notificationCenter.postNotificationName("LocationChange", object: self, userInfo: info)
+
+        UIAccessibilityPostNotification(UIAccessibilityScreenChangedNotification, nil)
```

Comment:
You're right the docs speak about a parameter. I tried passing the webview, but that selected the whole webview and cursor was stuck at it. On the other at that time I used just ScreenChanged and did not follow it with LayoutChanged (or the other way round). I will try to experiment with it a bit more and get back with results.

But it seems that passing `nil` is OK, as an Apple employee from accessibility team put it on the slides this way (see [slide 30 here](http://www.slideshare.net/cheinyeanlim/session-210-accessibilityforios)).